doc/bluealsa-cmd.1.rst

```diff
@@ -0,0 +1,111 @@
+============
+bluealsa-cmd
+============
+
+--------------------------------------------------------
+a simple command-line wrapper for the bluealsa D-Bus API
+--------------------------------------------------------
+
+:Date: February 2021
+:Manual section: 1
+:Manual group: General Commands Manual
+:Version: $VERSION$
+
+SYNOPSIS
+========
+
+**bluealsa-cmd** [*OPTION*]... *COMMAND* [*ARG*]...
+
+DESCRIPTION
+===========
+
+bluealsa-cmd provides command-line access to the BlueALSA D-Bus API. It
+implements the org.bluealsa.Manager1 and org.bluealsa.PCM1 interfaces and thus
+allows introspection and some control of BlueALSA PCMs while they are running.
+
+The *PCM_PATH* command argument, where required, must be a BlueALSA PCM D-Bus
+path.
+
+OPTIONS
+=======
+
+-h, --help
+    Output a usage message.
+
+-V, --version
+    Output the version number.
+
+-B NAME, --dbus=NAME
+    BlueALSA service name suffix. For more information see ``--dbus``
+    option of ``bluealsa(8)`` service daemon.
+
+-q, --quiet
+    Do not print any error messages.
+
+
+COMMANDS
+========
+
+get-codecs *PCM_PATH*
+    Print a list of additional codecs supported by the given PCM. For A2DP PCMs
+    this requires Bluez SEP support (bluez >= 5.52).
+
+list-pcms
+    Print a list of BlueALSA PCM D-Bus paths, one per line.
+
+monitor
+    Listen for ``PCMAdded`` and ``PCMRemoved`` signals and print a message on
+    standard output for each one received. Output lines are formed as
+    'PCMAdded *PCM_PATH*' or 'PCMRemoved *PCM_PATH*'.
+
+mute *PCM_PATH* y|n [y|n]
+    Set mute component of the volume property of the given PCM. The
+    second argument is used for stereo PCMs as described for ``set-volume``.
+
+open *PCM_PATH*
+    Connect to the given PCM then transfer audio frames. For sink PCMs
```

Comment:
I would say "raw audio frames" so it would be clear that "wav", "mp3" is not something that might be used here.

Reply:
changed in commit f013f26

Reply:
see commit 4b97de6